git/book

```diff
@@ -0,0 +1 @@
+/var/home/julian/Projekte/gnome/libraries/gtk4-rs/stable/latest/book
```

Comment:
That is probably not gonna work :p

Reply:
That is how I do it for the docs for each release
```
    - cd $repo
    - unlink ./stable/latest/docs
    - cd ./stable/latest
    - ln -sf ../$version/docs ./docs
 ```

Reply:
Mmmh weird, that's how I created the symbolic link as well.

Will have a look later.

Reply:
For some reason, I had to pass the `-r` flag for it to work.

Should be fine now.